.taskcluster.yml

```diff
@@ -147,12 +148,49 @@ tasks:
           owner: cdenizet@mozilla.com
           source: ${repository}/raw/${head_rev}/.taskcluster.yml
 
+      - $if: 'tasks_for == "github-push" && head_branch[:10] == "refs/tags/"'
+        then:
+          taskId: {$eval: as_slugid("test_mozilla_central")}
+          dependencies:
+            - {$eval: as_slugid("lint_test_task")}
+            - {$eval: as_slugid("windows_test_task")}
+          created: {$fromNow: ''}
+          deadline: {$fromNow: '1 hour'}
+          provisionerId: proj-relman
+          workerType: ci
+          payload:
+            maxRunTime: 3600
+            image: "rust:buster"
+            command:
+              - "/bin/bash"
+              - "-cx"
+              - "git clone --recursive --quiet ${repository} &&
+                 [ ! -d \"cache/gecko-dev\" ] &&
+                 git clone --quiet https://github.com/mozilla/gecko-dev.git /cache/gecko-dev || true &&
+                 pushd /cache/gecko-dev && git pull origin master && popd &&
+                 mkdir -p /tmp/mozilla_central_output &&
+                 cd rust-code-analysis &&
+                 git -c advice.detachedHead=false checkout ${head_rev} &&
+                 cargo build --release --all --all-features &&
+                 cargo run --release -p rust-code-analysis-cli -- -p /cache/gecko-dev \
+                                     -j24 --metrics -O json -o /tmp/mozilla_central_output"
+            cache:
+              rust-code-analysis-mozilla-central-repository: /cache
+          scopes:
+            - "docker-worker:cache:rust-code-analysis-mozilla-central-repository"
+          metadata:
+            name: rust-code-analysis mozilla-central test
+            description: rust-code-analysis-cli tested on mozilla-central
+            owner: cdenizet@mozilla.com
+            source: ${repository}/raw/${head_rev}/.taskcluster.yml
+
       - $if: 'tasks_for == "github-push" && head_branch[:10] == "refs/tags/"'
         then:
           taskId: {$eval: as_slugid("build_linux_release")}
           dependencies:
             - {$eval: as_slugid("lint_test_task")}
             - {$eval: as_slugid("windows_test_task")}
+            - {$eval: as_slugid("test_mozilla_central")}
```

Comment:
We can build the release if that test fails, let's make a dependency of the deployment task.

Reply:
I thought to add that dependency there to make the CI less heavy. If the `mozilla-central` task fails, we don't create the release binaries at all

Reply:
This test is so slow that the runtime of the others can be considered negligible. Better to catch all errors together, than fixing one, then releasing and noticing a second.

Reply:
Ok, I'm going to add that check within the deployment task